assets/js/src/index.js

```diff
@@ -355,12 +355,150 @@ function submitForm() {
     });
 }
 
-$('#prbotSubmit').click(function() {
+function submitFormSoftware() {
+  let submitButton = document.getElementById('prBotSubmitSoftware');
+  let resetButton = document.getElementById('softwareFormReset');
+  submitButton.disabled = true;
+  resetButton.disabled = true;
+
+  let content =
+    '' +
+    `---
+schemaVersion: "${$('#schemaVersion').val()}"
```

Comment:
It looks like you may have clobbered any changes that happened to master since you synced with it. This was the old way of writing the file.

Reply:
I am aware that this is the old version of doing things and that we're now using something along the getCodeObject() function. I made this code before this change was made and didn't wanted to update it right away. But not to worry, the getCodeObject() is still there, that snippet would be my separation of the code for the code and the software form so both are present.